### PR TITLE
add whether enabled & port number support for https

### PR DESCRIPTION
**Pull request by Cahb**

### CODE REVIEW

**Comment by Cahb** [comment]
Hey Jacky! Thanks for the changes, i've created a PR for a convinient discussion of these changes, if you don't mind;

We can discuss it here later on

File: ucentral.schema.json [hunk 1]
```diff
@@ -3634,6 +3634,20 @@
                 }
             }
         },
+        "service.https": {
+            "type": "object",
+            "properties": {
+                "https-port": {
+                    "type": "integer",
+                    "maximum": 65535,
+                    "minimum": 443,
```
Comment (mikehansen1970) [inline ucentral.schema.json:3643]: minimum for https should be 1
default for https should be 443

**Review by mikehansen1970 — CHANGES_REQUESTED** [review]
Please fix minimum and default for the ucentral.schema.json so that aligns with the pretty and full versions.
These schema files can, and should be generated using the generate.sh script in the repo which will ensure consistency.

**Comment by jacky1100002** [comment]
Thanks Mike. I will modify it according to your suggestion.

**Pull request merged**